less/pages/buyers-guide/shared.less

```diff
@@ -118,7 +122,7 @@
         background-image: url(/assets/buyers-guide/thumbnails/06-health-and-exercise-thumbnail.jpg);
       }
       &.why-link a {
-        background-image: url(/assets/buyers-guide/thumbnails/08-why-we-made-this-guide.jpg);
```

Comment:
Can you revert these changes here?

I think this is due to changes I made after you.

Reply:
Actually, this worries me a bit, that you may have accidentally reverted other things too.

Reply:
Things that are not immediately obvious.

Reply:
Just update these, and take a quick look through these changes and see if anything looks unfamiliar, otherwise I think we'll be good after the one change.